crates/utils/re_video/src/decode/gop_detection.rs

```diff
@@ -4,7 +4,13 @@ use h264_reader::{
     push::NalInterest,
 };
 
-use crate::{VideoCodec, VideoEncodingDetails, h264::encoding_details_from_h264_sps};
+use cros_codecs::codec::h265::parser::{Nalu, NaluType, Parser, Sps};
```

Comment:
nit: having both h264_reader's and cros_codec's symbols resolved here makes things confusing. The later could move into `detect_h265_annexb_gop` as they're only used there